File: src/classes/api/req_res/update-re-kyou-response.ts
// ˅
'use strict';

import { ReKyou } from '@/classes/datas/re-kyou';
import { GkillAPIResponse } from '../gkill-api-response';
import { Kyou } from '@/classes/datas/kyou';

// ˄

export class UpdateReKyouResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    updated_rekyou: ReKyou;

    updated_rekyou_kyou: Kyou;

    constructor() {
        // ˅
        super()
        this.updated_rekyou = new ReKyou()
        this.updated_rekyou_kyou = new Kyou()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/update_re_kyou_response.go
// ˅
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

// ˄

type UpdateReKyouResponse struct {
	// ˅

	// ˄

	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedReKyou *reps.ReKyou `json:"updated_rekyou"`

	UpdatedReKyouKyou *reps.Kyou `json:"updated_rekyou_kyou"`

	// ˅

	// ˄
}

// ˅

// ˄
